hono/.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
+tmp-d1-db
```

Comment:
It may be a good idea to specify the path `test/tmp-d1-db`

Reply:
Agreed.